torch_xla/csrc/lowering_context.cpp

```diff
@@ -75,14 +69,26 @@ xla::XlaOp LoweringContext::GetOutputOp(const Output& output) {
   return it->second;
 }
 
+void LoweringContext::LowerPostOrder(
+    tensorflow::gtl::ArraySlice<const Node* const> post_order) {
+  for (auto node : post_order) {
+    try {
+      node->Lower(this);
```

Comment:
When would `Node::Lower` throw an exception?

Reply:
Every time an XLA_* check fails within our lowering paths.

Reply:
Right, this makes sense.